docs/UserGuide.md

```diff
@@ -4,9 +4,9 @@
   pageNav: 3
 ---
 
-# AB-3 User Guide
+# VolunTier User Guide
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a  Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+VolunTier is a **desktop app for managing contacts, optimized for use via a  Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, VolunTier can get your contact management tasks done faster than traditional GUI apps.
```

Comment:
Could make it more specific to our target users and intended purpose

Reply:
Noted. Will make it more specific in future milestones. Thank you for your input!